AgentDiscoveries-Backend/src/main/java/org/softwire/training/api/routes/v1/UsersRoutes.java

```diff
@@ -36,6 +41,13 @@ public UserApiModel createEntity(Request req, Response res) throws FailedRequest
 
         int newUserId = usersDao.addUser(user);
 
+        //TODO creating new user should have choice to also create a corresponding agent.

```

Comment:
Does this correspond to a Trello card?

Reply:
I don't think so, but it goes with 
"Do we have a plan for what user/agent creation will actually look like (and a corresponding card)?"
To which I think the answer is approximately as agent creation is done in live at the moment:
http://ec2-35-178-54-146.eu-west-2.compute.amazonaws.com/#/admin/entities

Shall I create a card to make sure these two changes work together?

Reply:
If you'll do it as part of merging this, then no need. Otherwise yes please!